common/build/eslint-config-fluid/printed-configs/default.json

```diff
@@ -851,19 +886,271 @@
+            "error"
+        ],
+        "unicorn/prefer-dom-node-dataset": [
+            "error"
+        ],
+        "unicorn/prefer-dom-node-remove": [
+            "error"
+        ],
+        "unicorn/prefer-dom-node-text-content": [
+            "error"
+        ],
+        "unicorn/prefer-export-from": [
+            "error"
+        ],
+        "unicorn/prefer-includes": [
+            "error"
+        ],
+        "unicorn/prefer-json-parse-buffer": [
+            "error"
+        ],
+        "unicorn/prefer-keyboard-event-key": [
+            "error"
+        ],
+        "unicorn/prefer-math-trunc": [
+            "error"
+        ],
+        "unicorn/prefer-modern-dom-apis": [
+            "error"
+        ],
+        "unicorn/prefer-module": [
+            "error"
+        ],
+        "unicorn/prefer-negative-index": [
+            "error"
+        ],
+        "unicorn/prefer-node-protocol": [
+            "off"
+        ],
+        "unicorn/prefer-number-properties": [
+            "error"
+        ],
+        "unicorn/prefer-object-from-entries": [
+            "error"
+        ],
+        "unicorn/prefer-optional-catch-binding": [
+            "error"
+        ],
+        "unicorn/prefer-prototype-methods": [
+            "error"
+        ],
+        "unicorn/prefer-query-selector": [
+            "error"
+        ],
+        "unicorn/prefer-reflect-apply": [
+            "error"
+        ],
+        "unicorn/prefer-regexp-test": [
+            "error"
+        ],
+        "unicorn/prefer-set-has": [
+            "error"
+        ],
+        "unicorn/prefer-spread": [
+            "error"
+        ],
+        "unicorn/prefer-string-replace-all": [
+            "off"
+        ],
+        "unicorn/prefer-string-slice": [
+            "error"
+        ],
+        "unicorn/prefer-string-starts-ends-with": [
+            "error"
+        ],
+        "unicorn/prefer-string-trim-start-end": [
+            "error"
+        ],
+        "unicorn/prefer-switch": [
+            "error"
+        ],
+        "unicorn/prefer-ternary": [
+            "error"
+        ],
+        "unicorn/prefer-top-level-await": [
+            "off"
+        ],
+        "unicorn/prefer-type-error": [
+            "error"
+        ],
+        "unicorn/prevent-abbreviations": [
+            "off"
+        ],
+        "unicorn/relative-url-style": [
+            "error"
+        ],
+        "unicorn/require-array-join-separator": [
+            "error"
+        ],
+        "unicorn/require-number-to-fixed-digits-argument": [
+            "error"
+        ],
+        "unicorn/require-post-message-target-origin": [
+            "off"
+        ],
+        "unicorn/string-content": [
+            "off"
+        ],
+        "unicorn/template-indent": [
+            "warn"
+        ],
+        "unicorn/throw-new-error": [
+            "error"
+        ],
+        "use-isnan": [
+            "error"
+        ],
+        "valid-typeof": [
```

Comment:
valid-typeof seems like a good one to error on

Reply:
Made this an error.